addons/l10n_be_hr_payroll_fleet/models/fleet.py

```diff
@@ -66,6 +66,16 @@ def _compute_vehicle_name(self):
             acquisition_date = vehicle._get_acquisition_date()
             vehicle.name += u" \u2022 " + str(round(vehicle.total_depreciated_cost, 2)) + u" \u2022 " + acquisition_date
 
+    @api.model
+    def create(self, vals):
+        res = super(FleetVehicle, self).create(vals)
+        if not res.log_contracts:
+            self.env['fleet.vehicle.log.contract'].create({
+                'vehicle_id': res.id,
```

Comment:
Maybe check fields are correctly sync ? (at least driver otherwise current user is driver on the contract)